File: 2024/04 - April/22. Row with minimum no of 1's/Solution.py
class Solution:
    def minRow(self, n, m, a):
        res = float('inf')  
        ans = 0
        for i in range(n):
            row_sum = sum(1 for x in a[i] if x == 1)  
            if row_sum < res:
                res = row_sum
                ans = i
        return ans+1

File: 2024/04 - April/22. Row with minimum no of 1's/Solution.cpp
class Solution {
public:
    int minRow(int n, int m, vector<vector<int>>& a) 
    {
        int count = m, row = 0;
        for (int i = 0; i < n; i++) 
        {
            int res = 0; 
            for (int j = 0; j < m; j++) 
            {
                if (a[i][j] == 1) 
                {
                    res++;
                }
            }
            if (res < count) 
            {
                count = res;
                row = i;
            }
        }
        return row + 1;
    }
};
